src/components/GradientText/GradientText.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './GradientText.module.scss';
+
+const gradientText = ({ text, background }) => {
```

Comment:
minor, but the component's name should start with uppercase letter `gradientText` -> `GradientText `